experimental/ExtendTeamsforM365/client/pages/categories.js

```diff
@@ -14,9 +14,9 @@ async function displayUI() {
             categories.forEach(category => {
                 const categoryRow = document.createElement('tr');
                 categoryRow.innerHTML = `<tr>
-                <td><img src="data:image/bmp;base64,${category.picture}" class="categoryImage"></image></td>
+                <td id='categoriesImgae'><img src="data:image/bmp;base64,${category.picture}" class="categoryImage"></image></td>
```

Comment:
Is this is a typo `categoriesImgae` in the id @v-pankasahu ?

Reply:
Yeas, this was typo mistake. Corrected and renamed.